client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -14,7 +16,9 @@
                select-mode="single"
                @sort-changed="sortChanged"
                @row-selected="rowSelected">
-        <!-- A custom formatted column -->
+        <template v-slot:cell(number)="data">
+          <a @click="openOrderDetails(data.value)">{{ data.value }}</a>
```

Comment:
it does not look as link on view.

Reply:
Try to remove highlighting after row clicking (current selected row). It looks strange 
![image](https://user-images.githubusercontent.com/12604378/75506574-01d68000-5a11-11ea-9131-c20a233e54c6.png)